src/main/java/com/ss/ita/rozetka/ui/modals/ChangeTownModal.java

```diff
@@ -0,0 +1,37 @@
+package com.ss.ita.rozetka.ui.modals;
+
+import com.codeborne.selenide.SelenideElement;
+import com.ss.ita.rozetka.ui.pages.HomePage;
+import io.qameta.allure.Step;
+
+import static com.codeborne.selenide.Selenide.*;
+import static java.lang.String.format;
+
+public class ChangeTownModal {
```

Comment:
town or city?

Reply:
we can select all cities and towns in Ukraine